Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -341,6 +348,42 @@ def audit_formula_name
       openssl@1.1
     ].freeze
 
+    def audit_license
+      return unless @new_formula
+
+      if !formula.license.blank?
+        if @spdx_ids.key?(formula.license)
+          return unless @online
+
+          user, repo = get_repo_data(%r{https?://github\.com/([^/]+)/([^/]+)/?.*}, false)
+          return if user.nil?
+
+          github_license = GitHub.get_repo_license(user, repo)
+          return if github_license && (github_license == formula.license)
+
+          problem "License mismatch - Github license is: #{github_license}, "\
+          "but Formulae license states: #{formula.license}."
+        else
+          problem "#{formula.license} is not a standard SPDX license id."
+        end
+      else
+        problem "No license specified for package."
```

Comment:
```suggestion
      elsif @new_formula
        problem "No license specified for package."
```

Reply:
I think there shouldn't be a need for this as. I have a guard statement at the beginning of the method which is `return unless @new_formula`